build, commit: add support for disabling image history with `--add-history` (#4028)
containers/buildah: docs/buildah-build.1.md

## OPTIONS

**--add-history** *bool-value*

Adds build history for layers into the image spec. (default true).

> Following option is useful for the cases where end users explicitly

Reviewer: ```suggestion
This option is useful for the cases where end users explicitly
```

> wants to set `--add-history=false` to remove optional `History` from

Reviewer: ```suggestion
want to set `--add-history=false` to omit the optional `History` from
```

image-spec or when working with images built from build tools that

> does not honors setting `History` in image-spec.

Reviewer: ```suggestion
do not include `History` information in their images.
```
